include/tvm/relax/op_attr_types.h

```diff
@@ -80,6 +80,17 @@ using FCallPacked = String;
 using FTVMCompute = runtime::TypedPackedFunc<Array<te::Tensor>(
     const Attrs& attrs, const Array<te::Tensor>& inputs, const Type& out_type)>;
 
+/*!
+ * \brief Gradient for a specific op.
+ *        This definition is the same as tvm::relay::FPrimalGradient.
+ *
+ * \param orig_call the original Expr.
+ * \param output_grad the gradient of the Expr.
+ * \return the gradient for each parameters.
```

Comment:
`* \return the gradient for each parameter.` (maybe it is a typo in relay as well?)